skimage/measure/entropy.py

```diff
@@ -32,6 +33,8 @@ def shannon_entropy(image, base=2):
     --------
     >>> from skimage import data
     >>> shannon_entropy(data.camera())
-    17.732031303342747
+    4.8852702980667759
```

Comment:
Yes, indeed. 
The issue was that in my ipython console I calculated entropy with different bases, 
and 4.88... belongs to the base `e`, not the default 2. Fix has been sent.